apps/examples/load_fw/load_fw.c

```diff
@@ -8,6 +8,7 @@
  */
 
 #include <platform_info.h>
+#include <errno.h>
```

Comment:
for load_fw should be in common.h with other includes

Reply:
Done.